### ISSUE TITLE
how to evaluate adversarial model 

### ISSUE THREAD

**Issue by nvdang-dev** [liuyuemaicha/Adversarial-Learning-for-Neural-Dialogue-Generation-in-Tensorflow#20]
in code i see loss function with model G and D, but i want to evaluate evaluator reliability error (ERE) in paper. Thank you
